website/content/docs/concepts/security/data-encryption.mdx

```diff
@@ -20,7 +20,7 @@ Worker](/docs/configuration/worker/pki-worker) for storage of authentication
 keys. It is optional for PKI workers; if not specified the authentication keys
 will not be encrypted on disk. This is not used by KMS workers.
 
-## The `rootKey` KMS Key and Per-Scope KEK/DEKs <sup>OSS Only</sup>
+## The `root` KMS Key and Per-Scope KEK/DEKs <sup>OSS Only</sup>
```

Comment:
I changed this back to `root` since that is what is relevant for users configuring the controller (the `purpose` is `root`). `rootKey` is still returned on the API, "fixing" that would require a change in go-kms-wrapping.